index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en" xml:lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">  
+   
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    
+    <div class="flex_container backimage">
+    
+     <header class="header">
+
+     <menu class="topmenu">
+            <content class="ItemNavigation">
+                <content class="frame4">
+                    <content class="Portoflio">
+                        <img src="img/Portoflio.png" alt="Portfoliox">
+                        
+                    </content>
+                </content>
+                <content class="frame3">
+                    <content class="about">
+                        <img src="img/About.png" alt="Portfoliox">
+
+                        
+                    </content>
+                </content>
+                <content class="frame5">
+                    <content class="contact">
+                        <img src="img/Contact.png" alt="Portfoliox">
+
+                        
+                    </content>
+                </content>
+
+            </content>
+            
+           
+        </menu>
+     </header>
+     <main class="mainclass">
+        <header class="headlinclass" >
+
+            <content class="contenthead1">
+            <p> Hey there. I’m Alex.
+                I’m a software developer</p>
+            </content>
+            <section class="supportsection">
+                <p>  I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hesitate to contact me.</p>
+        
+            </section>
+            
+            <socialmedia class="socialmedia">    
+                <twitter class="twitter">
+                    <img src="img/twitter.png" alt="Network">  
+                </twitter>
+                <vectori class="vectori">
+                    <img src="img/vectori.png" alt="Network">  
+                </vectori>
+                <vectorn class="vectorin">
+                    <img src="img/vectorn.png" alt="Network">  
+                </vectorn></li>
+                <medium class="medium">
+                    <img src="img/medium.png" alt="Network">  
+                </medium></li>
+                 <github class="githubx">
+                    <img src="img/github.png" alt="Network">  
+                </github></li>
+                <angellist class="angellist">
+                    <img src="img/angellist.png" alt="Network">  
+                </angellist>
+            
+        
+            </socialmedia>
```

Comment:
- [ ] As requested by the previous reviewer, use `ul` and `li` to group the social media icons.  Each icon should also be a link.
 ```
 <ul>
    <li><a href="#">icon here </a></li>
    <li><a href="#">icon here </a></li>
    ............
 </ul>
 
 ```